moveit_ros/benchmarks/CMakeLists.txt

```diff
@@ -1,39 +1,47 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(moveit_ros_benchmarks)
 
+set(MOVEIT_LIB_NAME moveit_ros_benchmarks)
+
 add_compile_options(-std=c++11)
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED COMPONENTS date_time filesystem program_options system thread)
+find_package(Boost REQUIRED filesystem)
 
-find_package(catkin REQUIRED COMPONENTS
+find_package(catkin COMPONENTS
   moveit_ros_planning
   moveit_ros_warehouse
-  tf
-  eigen_conversions
   roscpp
-  rosconsole
+  REQUIRED
```

Comment:
according to https://cmake.org/cmake/help/v3.6/command/find_package.html the REQUIRED should remain before the COMPONENTS tag.